spec/controllers/v0/forms_controller_spec.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe V0::FormsController, type: :controller do
+  context 'with query param' do
+    it 'returns forms' do
+      VCR.use_cassette('forms/200_form_query') do
+        get :index, params: { query: 'health' }
+        expect(JSON.parse(response.body)['data'][0]['type']).to eq('va_form')
```

Comment:
You _could_ refactor the common bit of these two out

```
....
	expect type_of_response(response) to eq('va_form')
....
def type_of_response(response)
	JSON.parse(response.body)['data'][0]['type'])	
end
```

I think that then could be refactored out even further but that's a different issue 😄